src/cgame/cg_event.cpp

```diff
@@ -33,6 +33,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 CG_Obituary
 =============
 */
+// end on WHITE instead of ESC for player names following the tags
```

Comment:
Why?

Reply:
Just the cautious assumption, that if the default color of a message would be changed, that player and location names should maybe remain the same.
But i would have no problem with either way.

Might have to rename myself to 7T4im tho D:

Reply:
I mean, why did you make this assumption here but not anywhere else?

Reply:
Both versions, white-started names and neutral started names, are found throughout the code, i simply left most as they were.

Since players still have to be ported to CBSE, all those names have to be touched again; i suggest using that then to make it consistent via convenience method.

If you prefer the explicit white-prefixes removed till then, i'll gladly go through the code once more.

Reply:
There is a lot of assumptions everywhere, but perhaps we need to decide something in order to get coherency. Since the nicks are made to be colorful, we can assume _we have to_ set default colors when unspecified, i.e. white there. That's the same issue browser faces with white on white or dark on dark websites issues. That kind of issue can only be fixed with conventions, even if some people will disagree the convention chose something and not something else.

We have a similar issue with icons (= smileys), they are not displayed the same way throughout the ui. Basically there is some places where the selected color is applied on the icon itself, and sometime not. While the ability to color icons is not a bad idea by itself, we must define a a rule and stick to it. Not being able to color icons is not a bad loss btw, and it would avoid people to reset color when _they don't want_ to color icons, and who wants to color them?